src/generator/generate-types.ts

```diff
@@ -24,47 +24,62 @@ const unswitch = (o: any): any =>
         .reduce((a, b) => ({ ...a, ...b }), {})
     : o;
 
-const PathItemHack = (objName: string) => (o: any): any => ({
-  ...o,
-  definitions: {
-    ...o.definitions,
-    [objName]: {
-      ...o.definitions[objName],
-      properties: {
-        ...o.definitions[objName].properties,
-        ..."get|put|post|delete|options|head|patch|trace"
-          .split("|")
-          .map(i => ({
-            [i]:
-              o.definitions[objName].patternProperties[
-                "^(get|put|post|delete|options|head|patch|trace)$"
-              ]
-          }))
-          .reduce((a, b) => ({ ...a, ...b }), {})
+const HTTP_METHODS = [
+  "get",
+  "put",
+  "post",
+  "delete",
+  "options",
+  "head",
+  "patch",
+  "trace"
+];
+const HTTP_METHODS_REGEX = `^(${HTTP_METHODS.join("|")})$`;
+const PathItemHack = (objName: string) => (o: any): any => {
+  const pi = o.definitions[objName];
+  const opRef = pi.patternProperties[HTTP_METHODS_REGEX];
+  const methods = HTTP_METHODS.reduce(
+    (a, method) => ({ ...a, ...{ [method]: opRef } }),
+    {}
+  );
+  return {
```

Comment:
Could also use _lodash_ here (and in the rest of the hacks) - what do you think? @mikesol

Reply:
Lodash has massive security flaws on a regular basis, so I try to keep it to a strict minimum.

Reply:
That's fair. Are those `snyk`-reported security flaws?